template/bors.toml

```diff
@@ -6,5 +6,7 @@ status = [
     'Run cargo deny (bans licenses sources)'
 ]
 delete_merged_branches = true
+use_squash_merge = true
```

Comment:
We had issues where squash merges didn't close the PR properly.

I'm also curious about what use-case you're trying to solve with squashing? Git itself can show the history as if every merge was a squash with `git log --first-parent`.

Reply:
That was mostly to emulate how we did it before I thought maybe it was on accident that you missed that. I also kind of liked that all my "WIP" commits don't show up in the final thing but I'm happy either way

Reply:
I've removed it for now